vimeo/VimeoNetworking
# VIM-XXXX: Adds new Sign in with Apple requests

#### Ticket

TBC

#### Pull Request Checklist


#### Issue Summary

Adds Sign in with Apple request types to `VimeoNetworking`.

#### Implementation Summary

* Creates new `Request+Apple.swift` file with both login & join endpoints
* Accepts recommended settings warning for Pods project
* Moves all `Request` types into a `Requests` folder

The implementation of `Request+Apple.swift` largely mirrors the existing ones for Google and Facebook. The main difference is that validation from Apple requires 2 parameters - the user identity and an accompanying JSON web token which will be decoded and validated by the backend.

If successful this call returns a `VIMAccount`, otherwise it returns an error.

#### How to Test

There are unit tests that verify the requests have been created with the expected parameters. The endpoints used are not in place yet so there is no reliable way to test them yet.
